### PR TITLE
Add test of menu

### PR DESCRIPTION
**Pull request by xendk**
Basic test of menu.

Ref FWSP1-98

### CODE REVIEW

File: features/bootstrap/KKBasisContext.php [hunk 1]
```diff
@@ -188,13 +187,38 @@ public function iSetTheMainImageAlternativeTextTo($alt)
...
+    /**
+     * @When /^(?|I )name its menu item "((?:[^"]|\\")*)"$/
+     */
+    public function iNameItsMenuItem($name)
+    {
+        $this->getPage('NodeAdd')->setMenuTitle($this->fixStepArgument($name));
+    }
+
+    /**
+     * @Then /^"((?:[^"]|\\")*)" in the menu should point to the article I just created$/
```
Comment (naxoc) [inline features/bootstrap/KKBasisContext.php:204]: Is the trailing `$'` on purpose? In that case I don't understand it :)

Comment (xendk) [reply features/bootstrap/KKBasisContext.php:204]: Yes? It should match `Then "Ny artikel" in the menu should point to the article I just created`

Comment (naxoc) [reply features/bootstrap/KKBasisContext.php:204]: Arh. Nevermind. Too much morning here. :confused:  I thought the comment was out of the annotation.

File: features/bootstrap/Page/NodeAdd.php [hunk 2]
```diff
@@ -54,10 +74,30 @@ public function attachMainImage(string $path): string
         return $matches['imagepath'];
     }
 
+    /**
+     * Set the alt text for the main image.
```
Comment (googletorp) [inline features/bootstrap/Page/NodeAdd.php:78]: @xendk why no @param docs?

**Pull request merged**